src/sqlfluff/core/parser/markers.py

```diff
@@ -91,6 +91,31 @@ def from_point(
             **kwargs,
         )
 
+    @classmethod
+    def from_points(
+        cls,
+        start_point_marker: "PositionMarker",
+        end_point_marker: "PositionMarker",
+    ):
+        """Construct a position marker from the section between two points."""
+        return cls(
+            slice(
+                start_point_marker.source_slice.start,
+                end_point_marker.source_slice.stop,
+            ),
+            slice(
+                start_point_marker.templated_slice.start,
+                end_point_marker.templated_slice.stop,
+            ),
+            # The position markers should be the same so just pick
+            # the first.
```

Comment:
Which ones should be the same? What does "the first" mean?

Reply:
Will clarify....